### DISCUSSION TITLE
How to contact someone at Qfield?

### DISCUSSION THREAD

**Discussion by BTreeTech** [opengisch/QField#4692] [Q&A]
There are some improvements we would like and may well be able to sponsor. I have made contact via the forms on https://www.opengis.ch/ on two occasions with no response I have also tried linkedIn and X but I can't find anyone that will accept a message. 
Any tips ? @m-kuhn

**Comment by m-kuhn** [comment]
Hey @BTreeTech, sorry to hear you had troubles getting through. Can you send an email to info(at)opengis.ch (you can put matthias(at)opengis.ch in CC, just to be sure I get it)?
